src/callback_handler/kubernetes.rs

```diff
@@ -149,15 +147,19 @@ pub(crate) async fn has_list_resources_all_result_changed_since_instant(
 
 pub(crate) async fn can_i(
     client: Option<&mut Client>,
-    subject_access_review: &SubjectAccessReviewWrapper,
+    user: String,
+    group: String,
+    namespace: String,
+    resource: String,
+    verb: String,
```

Comment:
I've made a suggestion on the SDK about the type of payload we should send to the host. Given the new type is more complex, I would simply pass it as an argument (I think it should be doable, I hope I'm not missing something)